### PR TITLE
GameIndex: Add PSI's fix for Panzer Dragoon

### PR DESCRIPTION
**Pull request by knight-ryu12**
JayBee requested the pnach file and confirmed it worked.

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -21477,6 +21477,11 @@ Compat = 5
 Serial = SLPM-62718
 Name   = Sega Ages 2500 Series Vol.27 - Panzer Dragoon
 Region = NTSC-J
+[patches = 8D984276]
+	// By PSI
+	comment=Stop screen shaking (game switches to Progressive)
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: I prefer
```
comment=Patch by PSI
// Stop screen shaking (game switches to Progressive).
```

Comment (knight-ryu12) [reply bin/GameIndex.dbf]: I'd retain what patch actually is; not author

Comment (lightningterror) [reply bin/GameIndex.dbf]: All the other patches (most) are done in the format I suggested.

Comment (knight-ryu12) [reply bin/GameIndex.dbf]: I dislike that idea; Consistency is important, but I think it'd better to show what patch actually does.
Does it show both?

**Pull request merged**